### PR TITLE
plasma5: 5.23.1 -> 5.23.2

### PR DESCRIPTION
**Pull request by ttuegel**
###### Motivation for this change

This is a routine minor release.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 144124` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
110 packages marked as broken and skipped:
</summary>
<ul>
<li>libsForQt512.applet-window-buttons</li>
<li>libsForQt512.bluedevil</li>
<li>libsForQt512.breeze-grub</li>
<li>libsForQt512.breeze-gtk</li>
<li>libsForQt512.breeze-plymouth</li>
<li>libsForQt512.breeze-qt5</li>
<li>libsForQt512.discover</li>
<li>libsForQt512.kactivitymanagerd</li>
<li>libsForQt512.kde-cli-tools</li>
<li>libsForQt512.kde-gtk-config</li>
<li>libsForQt512.kde2-decoration</li>
<li>libsForQt512.kdecoration</li>
<li>libsForQt512.kdenlive</li>
<li>libsForQt512.kdeplasma-addons</li>
<li>libsForQt512.kgamma5</li>
<li>libsForQt512.khotkeys</li>
<li>libsForQt512.kinfocenter</li>
<li>libsForQt512.kmenuedit</li>
<li>libsForQt512.krohnkite</li>
<li>libsForQt512.kscreen</li>
<li>libsForQt512.kscreenlocker</li>
<li>libsForQt512.ksshaskpass</li>
<li>libsForQt512.ksystemstats</li>
<li>libsForQt512.kwallet-pam</li>
<li>libsForQt512.kwayland-integration</li>
<li>libsForQt512.kwayland-server</li>
<li>libsForQt512.kwin</li>
<li>libsForQt512.kwin-dynamic-workspaces</li>
<li>libsForQt512.kwin-tiling</li>
<li>libsForQt512.kwrited</li>
<li>libsForQt512.layer-shell-qt</li>
<li>libsForQt512.libkscreen</li>
<li>libsForQt512.libksysguard</li>
<li>libsForQt512.milou</li>
<li>libsForQt512.oxygen</li>
<li>libsForQt512.parachute</li>
<li>libsForQt512.plasma-browser-integration</li>
<li>libsForQt512.plasma-desktop</li>
<li>libsForQt512.plasma-disks</li>
<li>libsForQt512.plasma-integration</li>
<li>libsForQt512.plasma-nm</li>
<li>libsForQt512.plasma-pa</li>
<li>libsForQt512.plasma-sdk</li>
<li>libsForQt512.plasma-systemmonitor</li>
<li>libsForQt512.plasma-thunderbolt</li>
<li>libsForQt512.plasma-vault</li>
<li>libsForQt512.plasma-workspace</li>
<li>libsForQt512.plasma-workspace-wallpapers</li>
<li>libsForQt512.polkit-kde-agent</li>
<li>libsForQt512.powerdevil</li>
<li>libsForQt512.qqc2-breeze-style</li>
<li>libsForQt512.sddm-kcm</li>
<li>libsForQt512.spectacle</li>
<li>libsForQt512.systemsettings</li>
<li>libsForQt512.xdg-desktop-portal-kde</li>
<li>libsForQt514.applet-window-buttons</li>
<li>libsForQt514.bluedevil</li>
<li>libsForQt514.breeze-grub</li>
<li>libsForQt514.breeze-gtk</li>
<li>libsForQt514.breeze-plymouth</li>
<li>libsForQt514.breeze-qt5</li>
<li>libsForQt514.discover</li>
<li>libsForQt514.kactivitymanagerd</li>
<li>libsForQt514.kde-cli-tools</li>
<li>libsForQt514.kde-gtk-config</li>
<li>libsForQt514.kde2-decoration</li>
<li>libsForQt514.kdecoration</li>
<li>libsForQt514.kdenlive</li>
<li>libsForQt514.kdeplasma-addons</li>
<li>libsForQt514.kgamma5</li>
<li>libsForQt514.khotkeys</li>
<li>libsForQt514.kinfocenter</li>
<li>libsForQt514.kmenuedit</li>
<li>libsForQt514.krohnkite</li>
<li>libsForQt514.kscreen</li>
<li>libsForQt514.kscreenlocker</li>
<li>libsForQt514.ksshaskpass</li>
<li>libsForQt514.ksystemstats</li>
<li>libsForQt514.kwallet-pam</li>
<li>libsForQt514.kwayland-integration</li>
<li>libsForQt514.kwayland-server</li>
<li>libsForQt514.kwin</li>
<li>libsForQt514.kwin-dynamic-workspaces</li>
<li>libsForQt514.kwin-tiling</li>
<li>libsForQt514.kwrited</li>
<li>libsForQt514.layer-shell-qt</li>
<li>libsForQt514.libkscreen</li>
<li>libsForQt514.libksysguard</li>
<li>libsForQt514.milou</li>
<li>libsForQt514.oxygen</li>
<li>libsForQt514.parachute</li>
<li>libsForQt514.plasma-browser-integration</li>
<li>libsForQt514.plasma-desktop</li>
<li>libsForQt514.plasma-disks</li>
<li>libsForQt514.plasma-integration</li>
<li>libsForQt514.plasma-nm</li>
<li>libsForQt514.plasma-pa</li>
<li>libsForQt514.plasma-sdk</li>
<li>libsForQt514.plasma-systemmonitor</li>
<li>libsForQt514.plasma-thunderbolt</li>
<li>libsForQt514.plasma-vault</li>
<li>libsForQt514.plasma-workspace</li>
<li>libsForQt514.plasma-workspace-wallpapers</li>
<li>libsForQt514.polkit-kde-agent</li>
<li>libsForQt514.powerdevil</li>
<li>libsForQt514.qqc2-breeze-style</li>
<li>libsForQt514.sddm-kcm</li>
<li>libsForQt514.spectacle</li>
<li>libsForQt514.systemsettings</li>
<li>libsForQt514.xdg-desktop-portal-kde</li>
</ul>
</details>
<details>

  <summary>3 packages failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>index-fm: log was empty</li>
<li>libsForQt5.applet-window-buttons: log was empty</li>
<li>libsForQt5.plasma-vault: log was empty</li>
  </ul>
</details>
<details>

  <summary>54 packages failed to build and are new build failures:</summary>
  <ul>
<li>kdev-php: log was empty</li>
<li>kdev-python: log was empty</li>
<li>kdevelop: log was empty</li>
<li>kdevelop-unwrapped: log was empty</li>
<li>(libsForQt515.applet-window-buttons: log was empty</li>
<li>,plasma5Packages.applet-window-buttons): log was empty</li>
<li>libsForQt5.kde-cli-tools: log was empty</li>
<li>(libsForQt515.kde-cli-tools: log was empty</li>
<li>,plasma5Packages.kde-cli-tools): log was empty</li>
<li>libsForQt5.kdeplasma-addons: log was empty</li>
<li>(libsForQt515.kdeplasma-addons: log was empty</li>
<li>,plasma5Packages.kdeplasma-addons): log was empty</li>
<li>libsForQt5.khotkeys: log was empty</li>
<li>(libsForQt515.khotkeys: log was empty</li>
<li>,plasma5Packages.khotkeys): log was empty</li>
<li>libsForQt5.kinfocenter: log was empty</li>
<li>(libsForQt515.kinfocenter: log was empty</li>
<li>,plasma5Packages.kinfocenter): log was empty</li>
<li>libsForQt5.kmenuedit: log was empty</li>
<li>(libsForQt515.kmenuedit: log was empty</li>
<li>,plasma5Packages.kmenuedit): log was empty</li>
<li>libsForQt5.krohnkite: log was empty</li>
<li>(libsForQt515.krohnkite: log was empty</li>
<li>,plasma5Packages.krohnkite): log was empty</li>
<li>libsForQt5.kwin-dynamic-workspaces: log was empty</li>
<li>(libsForQt515.kwin-dynamic-workspaces: log was empty</li>
<li>,plasma5Packages.kwin-dynamic-workspaces): log was empty</li>
<li>libsForQt5.kwin-tiling: log was empty</li>
<li>(libsForQt515.kwin-tiling: log was empty</li>
<li>,plasma5Packages.kwin-tiling): log was empty</li>
<li>libsForQt5.parachute: log was empty</li>
<li>(libsForQt515.parachute: log was empty</li>
<li>,plasma5Packages.parachute): log was empty</li>
<li>libsForQt5.plasma-browser-integration: log was empty</li>
<li>(libsForQt515.plasma-browser-integration: log was empty</li>
<li>,plasma5Packages.plasma-browser-integration): log was empty</li>
<li>libsForQt5.plasma-desktop: log was empty</li>
<li>(libsForQt515.plasma-desktop: log was empty</li>
<li>,plasma5Packages.plasma-desktop): log was empty</li>
<li>libsForQt5.plasma-disks: log was empty</li>
<li>(libsForQt515.plasma-disks: log was empty</li>
<li>,plasma5Packages.plasma-disks): log was empty</li>
<li>(libsForQt515.plasma-vault: log was empty</li>
<li>,plasma5Packages.plasma-vault): log was empty</li>
<li>libsForQt5.plasma-workspace: log was empty</li>
<li>(libsForQt515.plasma-workspace: log was empty</li>
<li>,plasma5Packages.plasma-workspace): log was empty</li>
<li>libsForQt5.powerdevil: log was empty</li>
<li>(libsForQt515.powerdevil: log was empty</li>
<li>,plasma5Packages.powerdevil): log was empty</li>
<li>libsForQt5.systemsettings: log was empty</li>
<li>(libsForQt515.systemsettings: log was empty</li>
<li>,plasma5Packages.systemsettings): log was empty</li>
<li>wacomtablet: log was empty</li>
  </ul>
</details>
<details>
<summary>
42 packages built:
</summary>
<ul>
<li>digikam</li>
<li>haruna</li>
<li>libsForQt5.bluedevil (libsForQt515.bluedevil ,plasma5Packages.bluedevil)</li>
<li>libsForQt5.breeze-grub (libsForQt515.breeze-grub ,plasma5Packages.breeze-grub)</li>
<li>libsForQt5.breeze-gtk (libsForQt515.breeze-gtk ,plasma5Packages.breeze-gtk)</li>
<li>libsForQt5.breeze-plymouth (libsForQt515.breeze-plymouth ,plasma5Packages.breeze-plymouth)</li>
<li>libsForQt5.breeze-qt5 (libsForQt515.breeze-qt5 ,plasma5Packages.breeze-qt5)</li>
<li>libsForQt5.discover (libsForQt515.discover ,plasma5Packages.discover)</li>
<li>libsForQt5.kactivitymanagerd (libsForQt515.kactivitymanagerd ,plasma5Packages.kactivitymanagerd)</li>
<li>libsForQt5.kde-gtk-config (libsForQt515.kde-gtk-config ,plasma5Packages.kde-gtk-config)</li>
<li>libsForQt5.kde2-decoration (libsForQt515.kde2-decoration ,plasma5Packages.kde2-decoration)</li>
<li>libsForQt5.kdecoration (libsForQt515.kdecoration ,plasma5Packages.kdecoration)</li>
<li>libsForQt5.kdenlive (libsForQt515.kdenlive ,plasma5Packages.kdenlive)</li>
<li>libsForQt5.kgamma5 (libsForQt515.kgamma5 ,plasma5Packages.kgamma5)</li>
<li>libsForQt5.kscreen (libsForQt515.kscreen ,plasma5Packages.kscreen)</li>
<li>libsForQt5.kscreenlocker (libsForQt515.kscreenlocker ,plasma5Packages.kscreenlocker)</li>
<li>libsForQt5.ksshaskpass (libsForQt515.ksshaskpass ,plasma5Packages.ksshaskpass)</li>
<li>libsForQt5.ksystemstats (libsForQt515.ksystemstats ,plasma5Packages.ksystemstats)</li>
<li>libsForQt5.kwallet-pam (libsForQt515.kwallet-pam ,plasma5Packages.kwallet-pam)</li>
<li>libsForQt5.kwayland-integration (libsForQt515.kwayland-integration ,plasma5Packages.kwayland-integration)</li>
<li>libsForQt5.kwayland-server (libsForQt515.kwayland-server ,plasma5Packages.kwayland-server)</li>
<li>libsForQt5.kwin (libsForQt515.kwin ,plasma5Packages.kwin)</li>
<li>libsForQt5.kwrited (libsForQt515.kwrited ,plasma5Packages.kwrited)</li>
<li>libsForQt5.layer-shell-qt (libsForQt515.layer-shell-qt ,plasma5Packages.layer-shell-qt)</li>
<li>libsForQt5.libkscreen (libsForQt515.libkscreen ,plasma5Packages.libkscreen)</li>
<li>libsForQt5.libksysguard (libsForQt515.libksysguard ,plasma5Packages.libksysguard)</li>
<li>libsForQt5.milou (libsForQt515.milou ,plasma5Packages.milou)</li>
<li>libsForQt5.oxygen (libsForQt515.oxygen ,plasma5Packages.oxygen)</li>
<li>libsForQt5.plasma-integration (libsForQt515.plasma-integration ,plasma5Packages.plasma-integration)</li>
<li>libsForQt5.plasma-nm (libsForQt515.plasma-nm ,plasma5Packages.plasma-nm)</li>
<li>libsForQt5.plasma-pa (libsForQt515.plasma-pa ,plasma5Packages.plasma-pa)</li>
<li>libsForQt5.plasma-sdk (libsForQt515.plasma-sdk ,plasma5Packages.plasma-sdk)</li>
<li>libsForQt5.plasma-systemmonitor (libsForQt515.plasma-systemmonitor ,plasma5Packages.plasma-systemmonitor)</li>
<li>libsForQt5.plasma-thunderbolt (libsForQt515.plasma-thunderbolt ,plasma5Packages.plasma-thunderbolt)</li>
<li>libsForQt5.plasma-workspace-wallpapers (libsForQt515.plasma-workspace-wallpapers ,plasma5Packages.plasma-workspace-wallpapers)</li>
<li>libsForQt5.polkit-kde-agent (libsForQt515.polkit-kde-agent ,plasma5Packages.polkit-kde-agent)</li>
<li>libsForQt5.qqc2-breeze-style (libsForQt515.qqc2-breeze-style ,plasma5Packages.qqc2-breeze-style)</li>
<li>libsForQt5.sddm-kcm (libsForQt515.sddm-kcm ,plasma5Packages.sddm-kcm)</li>
<li>libsForQt5.spectacle (libsForQt515.spectacle ,plasma5Packages.spectacle)</li>
<li>libsForQt5.xdg-desktop-portal-kde (libsForQt515.xdg-desktop-portal-kde ,plasma5Packages.xdg-desktop-portal-kde)</li>
<li>lxqt.lxqt-config</li>
<li>plasma-applet-volumewin7mixer</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 144124` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
110 packages marked as broken and skipped:
</summary>
<ul>
<li>libsForQt512.applet-window-buttons</li>
<li>libsForQt512.bluedevil</li>
<li>libsForQt512.breeze-grub</li>
<li>libsForQt512.breeze-gtk</li>
<li>libsForQt512.breeze-plymouth</li>
<li>libsForQt512.breeze-qt5</li>
<li>libsForQt512.discover</li>
<li>libsForQt512.kactivitymanagerd</li>
<li>libsForQt512.kde-cli-tools</li>
<li>libsForQt512.kde-gtk-config</li>
<li>libsForQt512.kde2-decoration</li>
<li>libsForQt512.kdecoration</li>
<li>libsForQt512.kdenlive</li>
<li>libsForQt512.kdeplasma-addons</li>
<li>libsForQt512.kgamma5</li>
<li>libsForQt512.khotkeys</li>
<li>libsForQt512.kinfocenter</li>
<li>libsForQt512.kmenuedit</li>
<li>libsForQt512.krohnkite</li>
<li>libsForQt512.kscreen</li>
<li>libsForQt512.kscreenlocker</li>
<li>libsForQt512.ksshaskpass</li>
<li>libsForQt512.ksystemstats</li>
<li>libsForQt512.kwallet-pam</li>
<li>libsForQt512.kwayland-integration</li>
<li>libsForQt512.kwayland-server</li>
<li>libsForQt512.kwin</li>
<li>libsForQt512.kwin-dynamic-workspaces</li>
<li>libsForQt512.kwin-tiling</li>
<li>libsForQt512.kwrited</li>
<li>libsForQt512.layer-shell-qt</li>
<li>libsForQt512.libkscreen</li>
<li>libsForQt512.libksysguard</li>
<li>libsForQt512.milou</li>
<li>libsForQt512.oxygen</li>
<li>libsForQt512.parachute</li>
<li>libsForQt512.plasma-browser-integration</li>
<li>libsForQt512.plasma-desktop</li>
<li>libsForQt512.plasma-disks</li>
<li>libsForQt512.plasma-integration</li>
<li>libsForQt512.plasma-nm</li>
<li>libsForQt512.plasma-pa</li>
<li>libsForQt512.plasma-sdk</li>
<li>libsForQt512.plasma-systemmonitor</li>
<li>libsForQt512.plasma-thunderbolt</li>
<li>libsForQt512.plasma-vault</li>
<li>libsForQt512.plasma-workspace</li>
<li>libsForQt512.plasma-workspace-wallpapers</li>
<li>libsForQt512.polkit-kde-agent</li>
<li>libsForQt512.powerdevil</li>
<li>libsForQt512.qqc2-breeze-style</li>
<li>libsForQt512.sddm-kcm</li>
<li>libsForQt512.spectacle</li>
<li>libsForQt512.systemsettings</li>
<li>libsForQt512.xdg-desktop-portal-kde</li>
<li>libsForQt514.applet-window-buttons</li>
<li>libsForQt514.bluedevil</li>
<li>libsForQt514.breeze-grub</li>
<li>libsForQt514.breeze-gtk</li>
<li>libsForQt514.breeze-plymouth</li>
<li>libsForQt514.breeze-qt5</li>
<li>libsForQt514.discover</li>
<li>libsForQt514.kactivitymanagerd</li>
<li>libsForQt514.kde-cli-tools</li>
<li>libsForQt514.kde-gtk-config</li>
<li>libsForQt514.kde2-decoration</li>
<li>libsForQt514.kdecoration</li>
<li>libsForQt514.kdenlive</li>
<li>libsForQt514.kdeplasma-addons</li>
<li>libsForQt514.kgamma5</li>
<li>libsForQt514.khotkeys</li>
<li>libsForQt514.kinfocenter</li>
<li>libsForQt514.kmenuedit</li>
<li>libsForQt514.krohnkite</li>
<li>libsForQt514.kscreen</li>
<li>libsForQt514.kscreenlocker</li>
<li>libsForQt514.ksshaskpass</li>
<li>libsForQt514.ksystemstats</li>
<li>libsForQt514.kwallet-pam</li>
<li>libsForQt514.kwayland-integration</li>
<li>libsForQt514.kwayland-server</li>
<li>libsForQt514.kwin</li>
<li>libsForQt514.kwin-dynamic-workspaces</li>
<li>libsForQt514.kwin-tiling</li>
<li>libsForQt514.kwrited</li>
<li>libsForQt514.layer-shell-qt</li>
<li>libsForQt514.libkscreen</li>
<li>libsForQt514.libksysguard</li>
<li>libsForQt514.milou</li>
<li>libsForQt514.oxygen</li>
<li>libsForQt514.parachute</li>
<li>libsForQt514.plasma-browser-integration</li>
<li>libsForQt514.plasma-desktop</li>
<li>libsForQt514.plasma-disks</li>
<li>libsForQt514.plasma-integration</li>
<li>libsForQt514.plasma-nm</li>
<li>libsForQt514.plasma-pa</li>
<li>libsForQt514.plasma-sdk</li>
<li>libsForQt514.plasma-systemmonitor</li>
<li>libsForQt514.plasma-thunderbolt</li>
<li>libsForQt514.plasma-vault</li>
<li>libsForQt514.plasma-workspace</li>
<li>libsForQt514.plasma-workspace-wallpapers</li>
<li>libsForQt514.polkit-kde-agent</li>
<li>libsForQt514.powerdevil</li>
<li>libsForQt514.qqc2-breeze-style</li>
<li>libsForQt514.sddm-kcm</li>
<li>libsForQt514.spectacle</li>
<li>libsForQt514.systemsettings</li>
<li>libsForQt514.xdg-desktop-portal-kde</li>
</ul>
</details>
<details>

  <summary>3 packages failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>index-fm: log was empty</li>
<li>libsForQt5.applet-window-buttons: log was empty</li>
<li>libsForQt5.plasma-vault: log was empty</li>
  </ul>
</details>
<details>

  <summary>54 packages failed to build and are new build failures:</summary>
  <ul>
<li>kdev-php: log was empty</li>
<li>kdev-python: log was empty</li>
<li>kdevelop: log was empty</li>
<li>kdevelop-unwrapped: log was empty</li>
<li>(libsForQt515.applet-window-buttons: log was empty</li>
<li>,plasma5Packages.applet-window-buttons): log was empty</li>
<li>libsForQt5.kde-cli-tools: log was empty</li>
<li>(libsForQt515.kde-cli-tools: log was empty</li>
<li>,plasma5Packages.kde-cli-tools): log was empty</li>
<li>libsForQt5.kdeplasma-addons: log was empty</li>
<li>(libsForQt515.kdeplasma-addons: log was empty</li>
<li>,plasma5Packages.kdeplasma-addons): log was empty</li>
<li>libsForQt5.khotkeys: log was empty</li>
<li>(libsForQt515.khotkeys: log was empty</li>
<li>,plasma5Packages.khotkeys): log was empty</li>
<li>libsForQt5.kinfocenter: log was empty</li>
<li>(libsForQt515.kinfocenter: log was empty</li>
<li>,plasma5Packages.kinfocenter): log was empty</li>
<li>libsForQt5.kmenuedit: log was empty</li>
<li>(libsForQt515.kmenuedit: log was empty</li>
<li>,plasma5Packages.kmenuedit): log was empty</li>
<li>libsForQt5.krohnkite: log was empty</li>
<li>(libsForQt515.krohnkite: log was empty</li>
<li>,plasma5Packages.krohnkite): log was empty</li>
<li>libsForQt5.kwin-dynamic-workspaces: log was empty</li>
<li>(libsForQt515.kwin-dynamic-workspaces: log was empty</li>
<li>,plasma5Packages.kwin-dynamic-workspaces): log was empty</li>
<li>libsForQt5.kwin-tiling: log was empty</li>
<li>(libsForQt515.kwin-tiling: log was empty</li>
<li>,plasma5Packages.kwin-tiling): log was empty</li>
<li>libsForQt5.parachute: log was empty</li>
<li>(libsForQt515.parachute: log was empty</li>
<li>,plasma5Packages.parachute): log was empty</li>
<li>libsForQt5.plasma-browser-integration: log was empty</li>
<li>(libsForQt515.plasma-browser-integration: log was empty</li>
<li>,plasma5Packages.plasma-browser-integration): log was empty</li>
<li>libsForQt5.plasma-desktop: log was empty</li>
<li>(libsForQt515.plasma-desktop: log was empty</li>
<li>,plasma5Packages.plasma-desktop): log was empty</li>
<li>libsForQt5.plasma-disks: log was empty</li>
<li>(libsForQt515.plasma-disks: log was empty</li>
<li>,plasma5Packages.plasma-disks): log was empty</li>
<li>(libsForQt515.plasma-vault: log was empty</li>
<li>,plasma5Packages.plasma-vault): log was empty</li>
<li>libsForQt5.plasma-workspace: log was empty</li>
<li>(libsForQt515.plasma-workspace: log was empty</li>
<li>,plasma5Packages.plasma-workspace): log was empty</li>
<li>libsForQt5.powerdevil: log was empty</li>
<li>(libsForQt515.powerdevil: log was empty</li>
<li>,plasma5Packages.powerdevil): log was empty</li>
<li>libsForQt5.systemsettings: log was empty</li>
<li>(libsForQt515.systemsettings: log was empty</li>
<li>,plasma5Packages.systemsettings): log was empty</li>
<li>wacomtablet: log was empty</li>
  </ul>
</details>
<details>
<summary>
42 packages built:
</summary>
<ul>
<li>digikam</li>
<li>haruna</li>
<li>libsForQt5.bluedevil (libsForQt515.bluedevil ,plasma5Packages.bluedevil)</li>
<li>libsForQt5.breeze-grub (libsForQt515.breeze-grub ,plasma5Packages.breeze-grub)</li>
<li>libsForQt5.breeze-gtk (libsForQt515.breeze-gtk ,plasma5Packages.breeze-gtk)</li>
<li>libsForQt5.breeze-plymouth (libsForQt515.breeze-plymouth ,plasma5Packages.breeze-plymouth)</li>
<li>libsForQt5.breeze-qt5 (libsForQt515.breeze-qt5 ,plasma5Packages.breeze-qt5)</li>
<li>libsForQt5.discover (libsForQt515.discover ,plasma5Packages.discover)</li>
<li>libsForQt5.kactivitymanagerd (libsForQt515.kactivitymanagerd ,plasma5Packages.kactivitymanagerd)</li>
<li>libsForQt5.kde-gtk-config (libsForQt515.kde-gtk-config ,plasma5Packages.kde-gtk-config)</li>
<li>libsForQt5.kde2-decoration (libsForQt515.kde2-decoration ,plasma5Packages.kde2-decoration)</li>
<li>libsForQt5.kdecoration (libsForQt515.kdecoration ,plasma5Packages.kdecoration)</li>
<li>libsForQt5.kdenlive (libsForQt515.kdenlive ,plasma5Packages.kdenlive)</li>
<li>libsForQt5.kgamma5 (libsForQt515.kgamma5 ,plasma5Packages.kgamma5)</li>
<li>libsForQt5.kscreen (libsForQt515.kscreen ,plasma5Packages.kscreen)</li>
<li>libsForQt5.kscreenlocker (libsForQt515.kscreenlocker ,plasma5Packages.kscreenlocker)</li>
<li>libsForQt5.ksshaskpass (libsForQt515.ksshaskpass ,plasma5Packages.ksshaskpass)</li>
<li>libsForQt5.ksystemstats (libsForQt515.ksystemstats ,plasma5Packages.ksystemstats)</li>
<li>libsForQt5.kwallet-pam (libsForQt515.kwallet-pam ,plasma5Packages.kwallet-pam)</li>
<li>libsForQt5.kwayland-integration (libsForQt515.kwayland-integration ,plasma5Packages.kwayland-integration)</li>
<li>libsForQt5.kwayland-server (libsForQt515.kwayland-server ,plasma5Packages.kwayland-server)</li>
<li>libsForQt5.kwin (libsForQt515.kwin ,plasma5Packages.kwin)</li>
<li>libsForQt5.kwrited (libsForQt515.kwrited ,plasma5Packages.kwrited)</li>
<li>libsForQt5.layer-shell-qt (libsForQt515.layer-shell-qt ,plasma5Packages.layer-shell-qt)</li>
<li>libsForQt5.libkscreen (libsForQt515.libkscreen ,plasma5Packages.libkscreen)</li>
<li>libsForQt5.libksysguard (libsForQt515.libksysguard ,plasma5Packages.libksysguard)</li>
<li>libsForQt5.milou (libsForQt515.milou ,plasma5Packages.milou)</li>
<li>libsForQt5.oxygen (libsForQt515.oxygen ,plasma5Packages.oxygen)</li>
<li>libsForQt5.plasma-integration (libsForQt515.plasma-integration ,plasma5Packages.plasma-integration)</li>
<li>libsForQt5.plasma-nm (libsForQt515.plasma-nm ,plasma5Packages.plasma-nm)</li>
<li>libsForQt5.plasma-pa (libsForQt515.plasma-pa ,plasma5Packages.plasma-pa)</li>
<li>libsForQt5.plasma-sdk (libsForQt515.plasma-sdk ,plasma5Packages.plasma-sdk)</li>
<li>libsForQt5.plasma-systemmonitor (libsForQt515.plasma-systemmonitor ,plasma5Packages.plasma-systemmonitor)</li>
<li>libsForQt5.plasma-thunderbolt (libsForQt515.plasma-thunderbolt ,plasma5Packages.plasma-thunderbolt)</li>
<li>libsForQt5.plasma-workspace-wallpapers (libsForQt515.plasma-workspace-wallpapers ,plasma5Packages.plasma-workspace-wallpapers)</li>
<li>libsForQt5.polkit-kde-agent (libsForQt515.polkit-kde-agent ,plasma5Packages.polkit-kde-agent)</li>
<li>libsForQt5.qqc2-breeze-style (libsForQt515.qqc2-breeze-style ,plasma5Packages.qqc2-breeze-style)</li>
<li>libsForQt5.sddm-kcm (libsForQt515.sddm-kcm ,plasma5Packages.sddm-kcm)</li>
<li>libsForQt5.spectacle (libsForQt515.spectacle ,plasma5Packages.spectacle)</li>
<li>libsForQt5.xdg-desktop-portal-kde (libsForQt515.xdg-desktop-portal-kde ,plasma5Packages.xdg-desktop-portal-kde)</li>
<li>lxqt.lxqt-config</li>
<li>plasma-applet-volumewin7mixer</li>
</ul>
</details>

**Comment by andrevmatos** [comment]
I think f45e4ff331b267b433510446f71c2877afe40b44 needs to be reverted.

**Comment by ttuegel** [comment]
I'm still showing the NixOS test for Plasma hanging before reaching the desktop. I will investigate.

File: pkgs/desktops/plasma-5/plasma-workspace/default.nix [hunk 1]
```diff
@@ -53,12 +53,6 @@ mkDerivation {
     ./0001-startkde.patch
     ./0002-absolute-wallpaper-install-dir.patch
     ./0003-startkde-unit-detection.patch
-    # Included in 5.23.2
-    (fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/desktops/plasma-5/plasma-workspace/default.nix:57]: Please remove the unused fetchpatch from inputs.

**Comment by ttuegel** [comment]
I still haven't been able to fix the NixOS test failure. The Plasma session works fine (I'm using it right now). I suspect this has something to do with Wayland? I'm still looking.

**Comment by ttuegel** [comment]
The failure seems to be fixed by #143673.

**Pull request merged**